utils/terrascript_client.py

```diff
@@ -105,6 +107,18 @@ def __init__(self, integration, integration_prefix,
         self.uids = {a['name']: a['uid'] for a in filtered_accounts}
         self.default_regions = {a['name']: a['resourcesDefaultRegion']
                                 for a in filtered_accounts}
+        self.logtoes_zip = self.download_logtoes_zip(LOGTOES_RELEASE)
+
+    def download_logtoes_zip(self, release_url):
+        r = requests.get(GH_BASE_URL + '/' + release_url)
+        if r.status_code == 404:
+            raise ValueError(f"release is not found")
+        zip_url = r.json()['assets'][0]['browser_download_url']
+        zip_file = '/tmp/LogsToElasticsearch-'+r.json()['tag_name']+'.zip'
+        if not path.exists(zip_file):
+            r = requests.get(zip_url)
+            open(zip_file, 'wb').write(r.content)
```

Comment:
why do we need to use the local file system? can we keep the data in-memory?

Reply:
Tried but failed. Seems terraform only support local file. https://www.terraform.io/docs/providers/aws/r/lambda_function.html#filename

Reply:
ack